src/inference/dev_api/ie_system_conf.h

```diff
@@ -144,6 +159,22 @@ typedef enum {
     EFFICIENT_CORE_PROC = 3    //!< Processor based on Intel Efficient-cores
 } cpu_core_type_of_processor;
 
+/**
+ * @brief      Returns offset of cores
+ * @ingroup    ie_dev_api_system_conf
+ * @param[in]  core_type core type.
+ * @return     Offset of CPU cores.
+ */
+INFERENCE_ENGINE_API_CPP(int) getCoreOffset(const cpu_core_type_of_processor core_type);
+
+/**
+ * @brief      Returns step of threads
+ * @ingroup    ie_dev_api_system_conf
+ * @param[in]  core_type core type.
+ * @return     Step of threads.
+ */
+INFERENCE_ENGINE_API_CPP(int) getThreadStep(const cpu_core_type_of_processor core_type);
```

Comment:
add tests for newly add API here

Reply:
tests must be added in the PR where functionality is introduced. How can we guarantee that you don't introduce not-working functionality?

Reply:
@ilya-lavrenov As we discussed, test case for key logic will be handled in below items.
- Test case for CPU map parser on Linux is covered in [PR 15113](https://github.com/openvinotoolkit/openvino/pull/15113)
- Test case for CPU map parser on Windows will address in ticket 101223
- Test case for CPU map parser on Windows will address in ticket 101248